examples/bvh_driver/bvh_driver.cpp

```diff
@@ -245,6 +245,33 @@ class KokkosScopeGuard
       ->UseManualTime()                                                        \
       ->Unit(benchmark::kMicrosecond);
 
+class CmdLineArgs
+{
+  int argc_;
+  std::vector<char *> argv_;
+
+public:
+  CmdLineArgs(std::vector<std::string> const &args, char *exe)
+          : argc_(args.size() + 1), argv_{{exe}}
```

Comment:
NOTE: this is a bit sketchy because it assumes the executable name does not get out of scope...